azure-pipelines.yml

```diff
@@ -3,10 +3,15 @@ resources:
     - repository: templates
       type: github
       name: NSLS-II/profile-collection-ci
-      ref: refs/heads/main
+      # TODO: change it back to 'refs/heads/main' once
+      # https://github.com/NSLS-II/profile-collection-ci/pull/19 is merged.
+      ref: refs/heads/add-collection-2021-1.2
```

Comment:
Note: do not forget this TODO after https://github.com/NSLS-II/profile-collection-ci/pull/19 is merged to `main`.